.travis.yml

```diff
@@ -32,6 +32,10 @@ before_install:
         conda install --yes scipy matplotlib nose coverage setuptools;
         travis_retry pip install -q coveralls;
       fi;
+    # install AVbin
+    - wget https://github.com/downloads/AVbin/AVbin/install-avbin-linux-x86-64-v10 -O avbin.sh
```

Comment:
@Eric89GXL any idea why this `wget` command returns "4" (network error) all 3 times that travis tried it, but works fine locally?